xcode/Subconscious/Shared/Components/Detail/MemoEditorDetail.swift

```diff
@@ -189,6 +177,21 @@ struct MemoEditorDetailView: View {
         }
     }
     
+    private func blockEditor() -> some View {
+        BlockEditor.Representable(
+            store: blockEditorStore
+        )
+        .frame(
+            minHeight: UIFont.appTextMono.lineHeight * 8
+        )
+        .onReceive(
+            blockEditorStore.actions.compactMap(
+                MemoEditorDetailNotification.from
+            ),
+            perform: notify
+        )
```

Comment:
Replay select actions on MemoEditorDetail store. This allows us to `requestFindLinkDetail` by mapping from actions in block editor store.